src/frontend/src/pages/FinancePage/CreateReimbursementRequestForm/CreateReimbursementFormView.tsx

```diff
@@ -0,0 +1,230 @@
+    <>
+      <FormLabel>Receipts</FormLabel>
+      <ul>
+        {receiptFiles.map((receiptFile, index) => (
+          <li>
+            <Stack key={receiptFile.file.name} direction="row" justifyContent="space-between">
+              <Typography>{receiptFile.file.name}</Typography>
+              <IconButton onClick={() => receiptRemove(index)}>
+                <Delete />
+              </IconButton>
+            </Stack>
+          </li>
+        ))}
+      </ul>
+    </>
+  );
+
+  return (
+    <form
+      onSubmit={(e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        handleSubmit(onSubmit)(e);
+      }}
+    >
+      <PageLayout title="Create Reimbursement Request" previousPages={[]}>
+        <Grid container spacing={2}>
+          <Grid item container spacing={2} md={6} xs={12}>
+            <Grid item xs={12}>
+              <FormControl fullWidth>
+                <FormLabel>Purchased From</FormLabel>
+                <Controller
+                  name="vendorId"
+                  control={control}
+                  render={({ field: { onChange, value } }) => (
+                    <Select onChange={(newValue) => onChange(newValue.target.value)} value={value} error={!!errors.vendorId}>
+                      {allVendors.map((vendor) => (
+                        <MenuItem key={vendor.vendorId} value={vendor.vendorId}>
+                          {vendor.name}
+                        </MenuItem>
+                      ))}
+                    </Select>
+                  )}
+                />
+              </FormControl>
+              <FormHelperText>{errors.vendorId?.message}</FormHelperText>
+            </Grid>
+            <Grid item xs={6}>
+              <FormControl fullWidth>
+                <FormLabel>Refund Source</FormLabel>
+                <Controller
+                  name="account"
+                  control={control}
+                  render={({ field: { onChange, value } }) => (
+                    <Select
+                      onChange={(newValue) => onChange(newValue.target.value as ClubAccount)}
+                      value={value}
+                      error={!!errors.account}
+                    >
+                      {Object.values(ClubAccount).map((account) => (
+                        <MenuItem key={account} value={account}>
+                          {account}
+                        </MenuItem>
+                      ))}
+                    </Select>
+                  )}
+                />
+              </FormControl>
+              <FormHelperText>{errors.account?.message}</FormHelperText>
+            </Grid>
+            <Grid item xs={6}>
+              <FormControl fullWidth>
+                <FormLabel>Date of Expense</FormLabel>
+                <Controller
+                  name="dateOfExpense"
+                  control={control}
+                  render={({ field: { onChange, value } }) => (
+                    <DatePicker
```

Comment:
imo we should disable the ability to enter a date with the keyboard, only using the datepicker pop up makes it harder for a user to input wonky data

Reply:
do we do that anywhere else?

Reply:
no, that could be good easy tickets tho tbh

Reply:
what is it just disabling the textfield?

Reply:
probably